src/Vm.sol

```diff
@@ -31,133 +31,146 @@ interface VmSafe {
-    function envBytes32(string calldata) external view returns (bytes32);
-    function envString(string calldata) external view returns (string memory);
-    function envBytes(string calldata) external view returns (bytes memory);
-    // Reads environment variables as arrays, (name, delim) => (value[])
-    function envBool(string calldata, string calldata) external view returns (bool[] memory);
-    function envUint(string calldata, string calldata) external view returns (uint256[] memory);
-    function envInt(string calldata, string calldata) external view returns (int256[] memory);
-    function envAddress(string calldata, string calldata) external view returns (address[] memory);
-    function envBytes32(string calldata, string calldata) external view returns (bytes32[] memory);
-    function envString(string calldata, string calldata) external view returns (string[] memory);
-    function envBytes(string calldata, string calldata) external view returns (bytes[] memory);
-    // Read environment variables with default value, (name, value) => (value)
-    function envOr(string calldata, bool) external returns (bool);
-    function envOr(string calldata, uint256) external returns (uint256);
-    function envOr(string calldata, int256) external returns (int256);
-    function envOr(string calldata, address) external returns (address);
-    function envOr(string calldata, bytes32) external returns (bytes32);
-    function envOr(string calldata, string calldata) external returns (string memory);
-    function envOr(string calldata, bytes calldata) external returns (bytes memory);
+    function envBool(string calldata name) external view returns (bool);
+    function envUint(string calldata name) external view returns (uint256);
+    function envInt(string calldata name) external view returns (int256);
+    function envAddress(string calldata name) external view returns (address);
+    function envBytes32(string calldata name) external view returns (bytes32);
+    function envString(string calldata name) external view returns (string memory);
+    function envBytes(string calldata name) external view returns (bytes memory);
+    // Reads environment variables as arrays
+    function envBool(string calldata name, string calldata delim) external view returns (bool[] memory);
+    function envUint(string calldata name, string calldata delim) external view returns (uint256[] memory);
+    function envInt(string calldata name, string calldata delim) external view returns (int256[] memory);
+    function envAddress(string calldata name, string calldata delim) external view returns (address[] memory);
+    function envBytes32(string calldata name, string calldata delim) external view returns (bytes32[] memory);
+    function envString(string calldata name, string calldata delim) external view returns (string[] memory);
+    function envBytes(string calldata name, string calldata delim) external view returns (bytes[] memory);
```

Comment:
Do we want to give these a named return of `value` as we did with `envOr`?

Reply:
Yes. Actually they used to have the return value named, but I botched the rafactor.

Reply:
Fixed in the last commit.